_clients/javascript/index.md

```diff
@@ -141,6 +141,68 @@ async function search() {
 search().catch(console.log);
 ```
 
+## Authenticate with Amazon OpenSearch Service - AWS Sigv4
```

Comment:
Could we add a simple sentence to separate the headings? Suggest:

Use the following code to authenticate with Amazon OpenSearch Service.